app/views/admin/members/index.html.erb

```diff
@@ -1,3 +1,10 @@
+<div class="mt-5">
+  <%= search_form_for [:admin, @q], class: "form-inline my-2 my-lg-0"  do |f| %>
+    <%= f.label :first_name_or_email_cont, "Search :", class: "mr-2" %>
+    <%= f.search_field :first_name_or_email_cont, class: "form-control mr-sm-2" %>
+    <%= f.submit class: "btn btn-outline-success my-2 my-sm-0" %>
+  <% end %>
+</div>
```

Comment:
@vivekAI3927 there should be an input field for all attributes in member model, except created_at and updated_at